sandbox/src/main/scala/example/Sandbox.scala

```diff
@@ -209,6 +215,25 @@ object Sandbox extends TyrianApp[Msg, Model]:
             p(model.log.flatMap(msg => List(text(msg), br)))
           )
 
+        case Page.Page4 =>
+          val angle = model.currentTime.getSeconds() * 2 * math.Pi / 60 - math.Pi / 2
+          val handX = 50 + 40 * math.cos(angle)
+          val handY = 50 + 40 * math.sin(angle)
+          svg(attributes("viewBox" -> "0, 0, 100, 100", "width" -> "300px"))(
+            circle(
+              attributes("cx" -> "50", "cy" -> "50", "r" -> "45", "fill" -> "#0B79CE")
```

Comment:
The joy of eating your own dog food is that you get to see what sort of meal you made. 😄 
The `circle` tag looks perfect, but it looks like we'll need to generate the SVG attributes like `cx` too?

Reply:
Right, I thought about that. But I was a bit unsure how. I could add to the `HtmlAttributes` but that would mean you could apply them to any other tag, even if not circle or even svg? 
Would it make sense to also split the HtmlAttributes into SvgAttributes and only include those in the SVG object? It doesn't help with enforcing the types but could make it cleaner.

Reply:
I'm afraid that is probably the right thing to do yes. That way when you do the import you get the additional svg attributes.